// e8City
// Copyright (C) 2023 e8yes
//
// This program is free software: you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation, either version 3 of the License, or
// (at your option) any later version.
//
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU General Public License for more details.
//
// You should have received a copy of the GNU General Public License
// along with this program.  If not, see <http://www.gnu.org/licenses/>.

syntax = "proto3";

package e8;

// A 3D point in the Cartesian coordinate system.
message Point3 {
    float x = 1;
    float y = 2;
    float z = 3;
}

// A 3D vector in the Cartesian coordinate system.
message Vector3 {
    float x = 1;
    float y = 2;
    float z = 3;
}
